### PR TITLE
Test student show view

### PR DESCRIPTION
**Pull request by kurtisnelson**

### CODE REVIEW

File: spec/support/pundit.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module PunditViewPolicy
+  extend ActiveSupport::Concern
+
+  included do
+    before do
+      controller.singleton_class.class_eval do
+        def policy(instance)
+          Class.new do
+            def method_missing(*args, &block); true; end
```
Comment (jah2488) [inline spec/support/pundit.rb]: 😩 I'm assuming this is to mock out every pundit policy during tests to always return true?

Do we have to undo this if we want to write specs against the view and still have pundit do its thing?

File: spec/views/students/show.html.erb_spec.rb [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe "students/show", type: :view do
+  let(:student) { build_stubbed :student }
+  before do
+    render template: subject, locals: {
+      student: student.decorate
+    }
+  end
+  it "shows the student" do
+    expect(rendered).to match(/Stats/)
```
Comment (jah2488) [inline spec/views/students/show.html.erb_spec.rb]: I'm guessing this is just a smoke test to make sure the page actually renders?

**Comment by jah2488** [comment]
Ah, sorry @kurtisnelson I think my PR caused a conflict in the `students/show` template.

File: spec/support/pundit.rb (hunk 1 above)
Comment (kurtisnelson) [inline spec/support/pundit.rb]: I dug around and around the web on this one. But yes, this makes all view specs show everything. Without this, the `policy(model)` method is not found, which is worse.

File: spec/views/students/show.html.erb_spec.rb (hunk 2 above)
Comment (kurtisnelson) [inline spec/views/students/show.html.erb_spec.rb]: yep, it's because we have so many things that a student might not have filled out.

**Pull request merged**